processing-common/build.gradle.kts

```diff
@@ -1,4 +1,7 @@
 plugins {
+    id("com.livefront.sealedenum.kotlin")
+    id("com.livefront.sealedenum.detekt")
+    id("com.livefront.sealedenum.publish")
```

Comment:
So is the rough idea now that this is direct plugin relationship is being made explicit in each module, rather than relying on some of this happening in the base `build.gradle` file in a hidden way?

Reply:
Right, exactly. The convention plugins are composable in the sense that they contribute some additional configuration, which is opt-in by applying the convention plugin. In that way, build configuration is more declarative and less conditional.

With `subprojects` in a root `build.gradle`, it's all-or-nothing, unless that code starts becoming conditional based on which module it's being applied to.

If there's configuration unique to a single module, then that deserves to belong in just that module's `build.gradle`.

If there's a convention plugin that some module only wants to partially apply, then that convention plugin should be probably split up.